app/docs/0.13.x/proxy.md

```diff
@@ -934,70 +976,80 @@ HTTP/1.1 200 OK
 ...
 ```
 
-[Back to TOC](#table-of-contents)
+When establishing the connection and negotiating the SSL handshake, if your
+client sends `ssl-example.com` as part of the SNI extension, Kong will serve
+the `cert.pem` certificate previously configured.
 
-### The `https_only` property
-
-If you wish an API to only be served through HTTPS, you can do so by enabling
-its `https_only` property:
+[Back to TOC](#table-of-contents)
 
-```bash
-$ curl -i -X POST http://localhost:8001/apis \
-    -d "name=ssl-only-api" \
-    -d "upstream_url=http://example.com" \
-    -d "hosts=my-api.com" \
-    -d "https_only=true"
-HTTP/1.1 201 Created
-...
-```
+### Enforcing the client protocol (HTTP/HTTPS)
```

Comment:
suggestion: Enforcing -> Restricting

We can't force people to use a protocol, but we can restrict which ones we accept.